### DISCUSSION TITLE
Convert a DNG image to JPEG format

### DISCUSSION THREAD

**Discussion by lpmnobre** [dlemstra/Magick.NET#773] [Help] [answered]
![airport1](https://user-images.githubusercontent.com/32373070/97922741-b57bd680-1d3b-11eb-9ccf-dcff81f32c1d.jpg)
Could you help me please to configure a DNG image's conversion to JPEG **without changing the original colors**? 
The DNG file was downloaded from:
https://www.camerafv5.com/pages/raw-samples.php
image: airport1.dng

Here a snippet of my code:
```C#
MagickReadSettings settings = new MagickReadSettings { Format = MagickFormat.Dng,  };
var filestream = new FileStream(imageURL, FileMode.Open, FileAccess.Read);
using (filestream)
using (var image = settings == null ? new MagickImage(filestream) : new MagickImage(filestream, settings))
{
    image.Depth = 16;
    image.PhotonsAutoOrientation();
    image.Strip();
    image.Settings.SetDefines(new JpegWriteDefines()
    {
        Extent = (int)maxSizeInBytes / 1024
    } );
 }
```
Thanks so much!

**Comment by lpmnobre** [comment, marked as answer]
The problem related to the color background/brightness of the generated JPG was solved when I updated the Magick.Net and Ghostscript libraries.
I should have done this update before posting the issue. I'm sorry about that!

**Reply by dlemstra** [reply]
No problem, really appreciate that you came back to respond to your own question.

p.s. What does the `PhotonsAutoOrientation` extension method do?

**Comment by lpmnobre** [comment]
The PhotonsAutoOrientation method transforms an image with rotations and flip/flop operations, if necessary, to let it with the TopLeft orientation type.
I think the problem I reported was corrected in the 7.22 version (Remove incorrect GrayScale optimization for ToBitmap). 
Thanks so so much for your hard and brilliant work!
